code/modules/medical_machinery/machines/simple_medical_machinery.dm

```diff
@@ -0,0 +1,76 @@
+/obj/machinery/medical/simple
+	name = "Simple Medical Unit"
+	desc = "If you see this something went horrbily, horrbily wrong."
+	///What trait do we add while this is on
+	var/status
+	///What organ do we protect while this is on
+	var/organ_slot
+	///We remember how much heart support we actually provide
+	var/current_boost = 0
+	///We remember how much heart support we maximally provide
+	var/max_boost = 10
+	///Tracked damage of the organ
+	var/organ_damage
+
+/obj/machinery/medical/simple/clear_status()
+	. = ..()
+	REMOVE_TRAIT(attached,status,"mechanical")
+
+	if(!organ_slot)
+		return
+
+	attached.adjustOrganLoss(organ_slot,max_boost) //No spamming buckle unbuckle to heal instantly
```

Comment:
well, now you have the problem of people rapidly buckling and unbuckling themselves to these things to DEAL organ damage to themselves (10 organ damage per go)